query/outputnode.go

```diff
@@ -677,7 +677,7 @@ func valToBytes(v types.Val) ([]byte, error) {
 		return boolFalse, nil
 	case types.DateTimeID:
 		t := v.Value.(time.Time)
-		return t.MarshalJSON()
+		return marshalTimeJson(t)
```

Comment:
add comment at the top why we need this function, maybe link golang discussion from the go repo.

Reply:
Done